examples/mission-control/src/agent.rs

```diff
@@ -1,12 +1,13 @@
 use crate::account::*;
 use crate::asset::*;
 use crate::rate::*;
-use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::borsh::{BorshDeserialize, BorshSerialize};
 use near_sdk::serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
 #[derive(Clone, Serialize, Deserialize, BorshDeserialize, BorshSerialize)]
 #[serde(crate = "near_sdk::serde")]
+#[borsh(crate = "::near_sdk::borsh")]
```

Comment:
I would align these and I think `::` prefix is too verbose and could be confusing to users, so I suggest we drop it from the borsh attribute.

```suggestion
#[serde(crate = "near_sdk::serde")]
#[borsh(crate = "near_sdk::borsh")]
```